bindings/azure/blobstorage/blobstorage.go

```diff
@@ -135,9 +136,25 @@ func (a *AzureBlobStorage) Invoke(req *bindings.InvokeRequest) (*bindings.Invoke
 	}
 
 	// Unescape data which will still be a JSON string
-	unescapedData, _ := strconv.Unquote(string(req.Data))
+	unescapedData, unescapeError := strconv.Unquote(string(req.Data))
 
-	_, err := azblob.UploadBufferToBlockBlob(context.Background(), []byte(unescapedData), blobURL, azblob.UploadToBlockBlobOptions{
+	if unescapeError != nil {
+		return nil, unescapeError
+	}
+
+	var data []byte
+
+	if a.metadata.DecodeBase64 == "true" {
```

Comment:
Is `"true"` the only allowed positive value? What about other variations like `"True"`, etc?

Reply:
That's true. Currently, no other positive variations are valid. I have implemented it just like other components like [azure storagequeue](https://github.com/dapr/components-contrib/blob/04ce9b829946707fcf7abcb55b6a5ea6e749c86d/bindings/azure/storagequeues/storagequeues.go#L160) due to consistency. But I agree with you, that it could be more convenient to support other variations as well.

Reply:
You 're right. But what if I extend documentation for this component instead of commenting code? Which one would be better?

Reply:
Done!